packages/frontity-org-theme/src/index.ts

```diff
@@ -56,6 +57,8 @@ const frontityOrg: FrontityOrg = {
         ...borders,
         backgroundColor,
         textColor,
+        specialIcons,
```

Comment:
Hey, just before merging this PR, `specialIcons` is duplicated! It's also at the last position in the list. Please, remove one of them.

Reply:
whoops, fixed! :)